Core/Graphics/InternalInclude/Babylon/Graphics/FrameBuffer.h

```diff
@@ -65,6 +65,7 @@ namespace Babylon::Graphics
 
 
         bool m_disposed{false};
+        uintptr_t m_graphicsID;
```

Comment:
nit: multiple places
```suggestion
        uintptr_t m_deviceID;
```